x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.test.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import '../../../__mocks__/enterprise_search_url.mock';
+import { setMockValues } from '../../../__mocks__/kea.mock';
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { EuiButton, EuiCode } from '@elastic/eui';
+
+import { EmptyEngineOverview } from './engine_overview_empty';
+
+describe('EmptyEngineOverview', () => {
+  const values = {
+    engineName: 'empty-engine',
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockValues(values);
+  });
+
+  it('renders', () => {
+    const wrapper = shallow(<EmptyEngineOverview />);
+    expect(wrapper.find('h1').text()).toEqual('Engine Setup');
+    expect(wrapper.find('h2').text()).toEqual('Set up the “empty-engine” Engine');
+    expect(wrapper.find('h3').text()).toEqual('Indexing by API');
+  });
+
+  it('renders correctly versioned documentation URLs', () => {
+    const wrapper = shallow(<EmptyEngineOverview />);
+    expect(wrapper.find(EuiButton).prop('href')).toEqual(
+      expect.stringMatching(
+        /^(https:\/\/www\.elastic\.co\/guide\/en\/app-search\/)(\d\.\d)(\/index\.html)$/
+      )
+    );
```

Comment:
In hindsight I'm not 100% sure this was a good idea or worth writing a regex for lol, I guess I could do this instead?

```js
import { CURRENT_MAJOR } from '../../../../common/version';

// ...

    expect(wrapper.find(EuiButton).prop('href')).toEqual(
      `https://www.elastic.co/guide/en/app-search/${CURRENT_MAJOR}/index.html`
    );
```

🤷 what do people prefer?

Reply:
Ideally I would have loved to leave in, e.g., the `'8.0'` number statically but that would have crashed on the 7.x backport and we'd have to manually update it for 9.0.

Reply:
I prefer the later.

Reply:
👍 Will make the change!

Reply:
https://github.com/elastic/kibana/pull/83504/commits/5e945c372d784a1c9ed5bd88fea7abbf176e284f